### DISCUSSION TITLE
v1.03 no trackpad motion LED 6 unlit

### DISCUSSION THREAD

**Discussion by ian-thompson-yv** [george-norton/peacock#7] [Q&A] [answered]
Hi I have an assembled set of v1.03. The key switches and switch LEDs work but the pad is not responding and LED6 is unlit. I am getting 3.3V on the test point and 5V from VCC (3.3 from AVCC)

**Comment by george-norton** [comment]
Can you confirm you built firmware for peacock/v1_01? This is what you will get if you try to run the v1_00 firmware on a newer board.

**Reply by george-norton** [reply]
LED6 is an indicator LED, the default keymaps should turn it on if you switch layers.

**Reply by ian-thompson-yv** [reply]
I have built the default target in `peacock` branch of your forked QMK

**Reply by george-norton** [reply, marked as answer]
So, the correct keymap is default. But there are two different versions of the PCB - the first version has a mistake which requires bodge wires. You should be building with a command like:

`make georgen/peacock/v1_01:default
qmk compile -kb georgen/peacock/v1_01 -km default`

Where the keyboard is `georgen/peacock/v1_01` and the keymap is default.

**Reply by ian-thompson-yv** [reply]
Cool that works off the `multitouch_experiment` branch. The version isn't defined on `peacock`
Can I quickly check if more vinyl layers reduces twitchiness or if that should be only tuned via the threshold?

**Reply by george-norton** [reply]
In my testing you want at least 3 layers of Vinyl to get it tracking well. I have a work in progress debug application which should make tuning easier. I plan on having another go at tuning the sensor once that is all working.

**Reply by ian-thompson-yv** [reply]
I think I got to 5 or 6 trying to see if it helped 😆

**Reply by ian-thompson-yv** [reply]
If you want a MacOs tester I'm happy to help
